examples/trading-runner/src/index.ts

```diff
@@ -0,0 +1,537 @@
+      throw new Error(`flags must begin with '--': ${arg}`);
+    }
+
+    const parts = arg.slice(2).split("=");
+    if (parts.length != 2) {
+      throw new Error(`flags must have syntax '--key=value': ${arg}`);
+    }
+
+    const key = parts[0];
+    const value = parts[1];
+
+    if (!key) {
+      throw new Error(`invalid flags provided`);
+    }
+    options[key] = value;
+  }
+
+  // overwrite key env vars
+  if (isKeyOfObject("key", options)) {
+    let keyName = options["key"];
+
+    if (!isKeyOfObject(keyName, keys)) {
+      throw new Error(`no key defined with name: ${keyName}`);
+    }
+
+    process.env.API_PUBLIC_KEY = keys[keyName]!.publicKey;
+    process.env.API_PRIVATE_KEY = keys[keyName]!.privateKey;
+  }
+
+  const commands: { [key: string]: Function } = {
+    setup: setup,
+    trade: trade,
+    sweep: sweep,
+  };
+
+  if (!isKeyOfObject(command!, commands)) {
+    throw new Error(`Unknown command: ${command}`);
+  }
+
+  commands[command]!(options);
+}
+
+main().catch((error) => {
+  console.error(error);
+  process.exit(1);
+});
+
+async function setup(_options: any) {
+  // setup user tags
+  const adminTagId = await createUserTag("Admin", []);
+  const traderTagId = await createUserTag("Trader", []);
+
+  // setup users
+  await createUser("Alice", [adminTagId], "Alice key", keys!.alice!.publicKey!);
+  await createUser("Bob", [traderTagId], "Bob key", keys!.bob!.publicKey!);
+
+  // setup private key tags
+  const tradingTagId = await createPrivateKeyTag("trading", []);
+  const personal = await createPrivateKeyTag("personal", []);
+  const longTermStorageTagId = await createPrivateKeyTag(
+    "long-term-storage",
+    []
+  );
+
+  // setup private keys
+  await createPrivateKey("Trading Wallet", [tradingTagId]);
+  await createPrivateKey("Long Term Storage", [longTermStorageTagId]);
+  await createPrivateKey("Personal", [personal]);
+
+  // setup policies: grant specific users permissions to use specific private keys
+  // ADMIN
+  await createPolicy(
+    "Admin users can do everything",
+    "EFFECT_ALLOW",
+    `approvers.any(user, user.tags.contains('${adminTagId}'))`,
+    "true"
+  );
+
+  const paddedRouterAddress = SWAP_ROUTER_ADDRESS.toLowerCase()
+    .substring(2)
+    .padStart(64, "0");
+
+  // TRADING
+  await createPolicy(
+    "Traders can use trading keys to deposit, aka wrap, ETH",
+    "EFFECT_ALLOW",
+    `approvers.filter(user, user.tags.contains('${traderTagId}')).count() >= 1`,
+    `private_key.tags.contains('${tradingTagId}') && eth.tx.to == '${WETH_TOKEN_GOERLI.address}' && eth.tx.data[0..10] == '${DEPOSIT_SIGNATURE}'`
+  );
+  await createPolicy(
+    "Traders can use trading keys to withdraw, aka unwrap, WETH",
+    "EFFECT_ALLOW",
+    `approvers.filter(user, user.tags.contains('${traderTagId}')).count() >= 1`,
+    `private_key.tags.contains('${tradingTagId}') && eth.tx.to == '${WETH_TOKEN_GOERLI.address}' && eth.tx.data[0..10] == '${DEPOSIT_SIGNATURE}'`
```

Comment:
should this be `WITHDRAW_SIGNATURE`?

Also, you could combine some of these policies if you'd like:
```
private_key.tags.contains('${tradingTagId}') && eth.tx.to == '${WETH_TOKEN_GOERLI.address}' && eth.tx.data[0..10] in ['${DEPOSIT_SIGNATURE}', '${WITHDRAW_SIGNATURE}']
```

Reply:
probably easier to understand split, but wanted to show that it IS possible

Reply:
ended up adding it in the readme https://github.com/tkhq/sdk/pull/91/commits/2e9afe073b690c2a5d0debcd104c20b023d4373b